### DISCUSSION TITLE
Create relative path links

### DISCUSSION THREAD

**Discussion by Etienne-TR** [Zettlr/Zettlr#3556] [Ideas]
Relative path links can be done actually by 2 methods : 
1. Drag and drop file from OS file explorer to Zettlr editor
2. Drag and drop file from the attachments Zettlr explorer (other file in the right sidebar)

I would like to make this kind of link to markdown file inside Zettlr. The most similar existing action is `Copy full path` in edition or tab context menus and then by creating new link by the link button or by CTRL+K.
The path is automaticaly paste to link part syntaxe like this : `[](/full-path/file-name.md)`.

I wish there is a `"Copy relative link"` action in the edition and tab context menus that could be used to autofill a new link like this : `[name or title file](./relative-path/file-name.md)`

Seems to be useful to other than me ?

**Comment by jefferys** [comment]
The difficulty of creating relative path links is a blocker that prevents me from switching from Obsidian. Markdown with relative-path links is the best compromise for portability and external tool parsing. Why do I want to put significant long-term effort into an easily readable base file format, and then use non-human readable links, with a one-point-of-failure database that if corrupted ruins potentially years of work?
